webpack.common.ts

```diff
@@ -155,7 +155,9 @@ module.exports = {
       filename: `${STATIC_DIRECTORY}[contenthash:10].css`,
       chunkFilename: `${STATIC_DIRECTORY}[id].[contenthash:10].css`,
     }),
-    new WorkerPlugin(),
+    new WorkerPlugin({
+      filename: `${STATIC_DIRECTORY}[name].worker.[contenthash].js`,
```

Comment:
Handle worker conflict (with csv worker) when hmr renames them both as `0.worker.js`
<img width="350" alt="Screen Shot 2022-05-11 at 3 10 11 PM" src="https://user-images.githubusercontent.com/10232835/167985201-35e7c399-5e54-449b-94ce-bf45c14c528f.png">

Now have:

<img width="350" alt="Screen Shot 2022-05-11 at 3 16 21 PM" src="https://user-images.githubusercontent.com/10232835/167985251-44addf6d-af37-46c9-aec4-90f4fb1f7471.png">

Reply:
that's real annoying!